Directory.Build.props

```diff
@@ -103,26 +103,21 @@
 
   <!-- Common repo directories -->
   <PropertyGroup>
-    <!-- Properties to remove with buildtools removal -->
-    <ProjectDir>$(RepoRoot)</ProjectDir>
-    <BinDir>$(ProjectDir)artifacts\bin\</BinDir>
-    <ToolsDir Condition="'$(ToolsDir)'==''">$(ProjectDir)Tools/</ToolsDir>
-    <BuildToolsTaskDir Condition="'$(MSBuildRuntimeType)' != 'core'">$(ToolsDir)net46/</BuildToolsTaskDir>
-    <BuildToolsTaskDir Condition="'$(MSBuildRuntimeType)' == 'core'">$(ToolsDir)</BuildToolsTaskDir>
+    <ArtifactsBinDir>$(RepoRoot)artifacts\bin\</ArtifactsBinDir>
```

Comment:
Doesn't this property get defined identically by Arcade?

Reply:
Yes, but we need it earlier than arcade defines it because we have to define some paths before arcade props get imported.  It existed in a similar way before but I named them the same so it would be obvious.